ci-operator/jobs/coreos/rpm-ostree/OWNERS

```diff
@@ -0,0 +1,14 @@
+approvers:
```

Comment:
These need `reviewers` so that we could LGTM PRs here

Reply:
Aren't approvers also reviewers?

Reply:
No, these are different - https://github.com/kubernetes/test-infra/tree/master/prow/plugins/approve/approvers#overview

Reply:
Hmm but how does e.g. https://github.com/openshift/machine-config-operator/blob/master/OWNERS and https://github.com/openshift/cluster-version-operator/blob/master/OWNERS etc. work then?

Reply:
Oh, it seems it works like this too

Reply:
Can you `/lgtm` this?